frontend/public/models/index.ts

```diff
@@ -1260,3 +1260,23 @@ export const ConsolePluginModel: K8sKind = {
   labelPluralKey: 'public~ConsolePlugins',
   crd: true,
 };
+
+export const CloudCredentialModel: K8sKind = {
+  kind: 'CloudCredential',
+  label: 'CloudCredential',
+  labelPlural: 'CloudCredentials',
+  apiGroup: 'operator.openshift.io',
+  apiVersion: 'v1',
+  abbr: 'CO',
+  plural: 'cloudcredentials',
+};
+
+export const AuthenticationModel: K8sKind = {
+  kind: 'Authentication',
+  label: 'Authentication',
+  labelPlural: 'Authentications',
+  apiGroup: 'config.openshift.io',
+  apiVersion: 'v1',
+  plural: 'authentications',
+  abbr: 'AU',
+}
```

Comment:
```suggestion
};
```
Lint error.  Note you can test for this locally using `yarn lint`.  Or if you're using VS Code as your editor, the lint fixing will occur on save.

Reply:
there's a pre-commit linter that runs - assumed (incorrectly) that that was it... will do